guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -82,39 +71,50 @@ For more information about audience support, see https://access.redhat.com/docum
 +
 * From the *Mapper Type* list, select *Group Membership*.
 +
-* From the *Token Claim Name* list, select *groups*.
+* In *Token Claim Name*, enter *groups*.
 +
 * Set the *Full group path* toggle to *OFF*.
 +
 For more information about group mappers, see https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/server_administration_guide/user-storage-federation#ldap_mappers[Group Mapper] in the _LDAP Mappers_ section of the _{Keycloak} Server Administration Guide_.
 
 .. Click *Save*.
-
-. In the {Project} web UI, navigate to *Administer* > *Settings*, and click the *Authentication* tab.
+.. Navigate to *Groups*, and click *New*.
++
+* Enter a *Name* for the group.
+* Click *Save*.
+.. Navigate to *Users* and click *Add user*. 
++
+* Enter the *Username*, *Email*, and other user details. Set the *User Enabled* toggle to *ON*.
++
+* Click *Save*. The user is now created.
++
+* Click the *Credentials* tab and reset the user password under *Manage Password*: Enter *New Password* and confirm the password, set the *Temporary* toggle to *OFF*, and click *Reset Password*.
++
+* Click the *Groups* tab. Select the group you want to join from the *Available Groups* list, and click *Join*. The user is now part of the selected group.
+. In the {Project} web UI, navigate to *Administer* > *User Groups*.
+. Click *Create User Group*, and enter a *Name* for the user group.
+. Click the *Roles* tab, and assign appropriate roles to the user group.
+. Click the *External Groups* tab, then click *Add external user group*.
+. Enter the group name created in the {Keycloak} web UI.
+. Click *Submit*.
+. Navigate to *Administer* > *Settings*, and click the *Authentication* tab.
 . Locate the *Authorize login delegation* row, and in the *Value* column, set the value to `Yes`.
 . Locate the *Authorize login delegation auth source user autocreate* row, and in the *Value* column, set the value to `External`.
 . Locate the *Login delegation logout URL* row, and in the *Value* column, set the value to `https://{foreman-example-com}/users/extlogout`.
 +
 For the following steps, you can retrieve the values that you require by navigating to the following URL:  `_{Keycloak-short}.example.com_/auth/realms/_{Keycloak-short}_REALM_/.well-known/openid-configuration`.
 +
 . Locate the *OIDC Algorithm* row, and in the *Value* column, set the algorithm for encoding on {Keycloak}, for example, `RS256`.
-. Locate the *OIDC Audience* row, and in the *Value*  column, set the value to the client ID for {Keycloak}: `['_{foreman-example-com}_']`.
+. Locate the *OIDC Audience* row, and in the *Value*  column, set the value to the client ID for {Keycloak}: `['_{foreman-example-com}_-foreman-openidc}_']`.
```

Comment:
This expands to `['satellite.example.com-foreman-openidc}_']`, seems like a curly bracket is missing.

Reply:
Fixed. The second curly bracket was in fact not required, so removed it.